DeepSpeech.py

```diff
@@ -717,7 +718,8 @@ def calculate_report(results_tuple):
     items = list(zip(*results_tuple))
     mean_wer = 0.0
     for label, decoding, distance, loss in items:
-        corrected = correction(decoding, alphabet)
+        # corrected = correction(decoding, alphabet)
```

Comment:
Could you completely remove this? History can keep it for us.

Reply:
This was a left over from testing, I undid it. As for removing the correction call/spell.py, I'm leaving that for the PR that integrates the LM into the decoder.

Reply:
OK, make sense